cypress/e2e/internationalization.cy.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-undef */
+describe("Internationalization Tests", () => {
+	it("should use default English (en-BR) language when internationalization property is empty", () => {
```

Comment:
If I'm correct, it should be `en-GB` or `en-US` instead: https://developer.mozilla.org/en-US/docs/Glossary/BCP_47_language_tag